PlateSpace/ETL/index.js

```diff
@@ -51,8 +51,6 @@ var restSchema = mongoose.Schema({
   numberOfRates: Number
 });
 
-restSchema.index({ location: '2dsphere' });
```

Comment:
definitely want to keep this here, and perhaps add the unique one as Mike suggested